nixos/modules/services/torrent/transmission.nix

```diff
@@ -182,7 +182,7 @@ in
         example = "770";
```

Comment:
I don't think this file is supposed to be part of this PR :-)

Reply:
Thanks, yeah that got in by accident. I'll rebase and push soon.